components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/application/management/v1/ApplicationModel.java

```diff
@@ -41,6 +41,7 @@
     private String imageUrl;
     private String accessUrl;
     private String templateId;
+    private Boolean isManagementApp;
```

Comment:
is using Boolean instead of primitive an intended one? if so why was that?

Reply:
It is from the autogenerated code from swagger definition. We have the same Boolean used in other places. When we create the framework Service Provider Object, the null check is handled.